docs/source/tutorials/projects.rst

```diff
@@ -0,0 +1,1213 @@
+* ``shape`` the shape of the object, which is an :class:`.ObjectShape` enum.
+
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. literalinclude:: /code_examples/tutorials/projects/adding_objects_to_a_project_ontology.py
+            :language: python
+
+    .. tab:: Example output
+
+        .. code-block:: python
+
+            True  # False if unsuccessful
+
+
+Labels
+======
+
+.. note::
+    This section is undergoing a reconstruction [Tuesday, May 10th 2022].
+
+A label row has the attributes ``<label_hash>`` uid, ``data_title``, ``data_type``, ``data_units``, and ``label_status``.
+
+.. code-block::
+
+    {
+        'label_hash': label_hash (uid),
+        'data_title': data_title,
+        'data_type': data_type,
+        'data_units': data_units,
+        'object_answers': object_answers,
+        'classification_answers': classification_answers,
+        'label_status': label_status,
+    }
+
+A label row contains a data unit or a collection of data units and associated labels, and is specific to a data asset with type video or image group.
+
+1.  A label row with a data asset of type ``video`` contains a single data unit.
+2.  A label row with a data asset of type ``img_group`` contains any number of data units.
+
+A data unit can have any number of vector labels (e.g. bounding box, polygon, keypoint) and classifications.
+
+Before you start, make sure that a project client is initialised with the appropriate project ID and API key.
+
+.. code-block::
+
+    from encord.client import EncordClient
+
+    client = EncordClient.initialise("<project_id>", "<project_api_key>")
+
+
+Getting label rows
+------------------
+
+Project label row IDs (``<label_hash>`` uid) are found in the project information Python JSON (``client.get_project()``), which also contain information about the data title (``data_title``), data type (``data_type``) and label status (``label_status``).
+
+.. code-block::
+
+    {
+        ...,
+        'label_rows': [{
+            'label_hash': 'sample_label_uid',
+            'data_hash': 'sample_data_uid',
+            'dataset_hash': 'sample_dataset_uid',
+            'dataset_title': 'sample_dataset_title',
+            'data_title': 'sample_data.mp4',
+            'data_type': 'VIDEO',
+            'label_status': 'LABELLED',
+        }, {
+            'label_hash': ...,
+        }]
+    }
+
+
+Use the client to fetch individual label objects.
+
+.. code-block::
+
+    label = client.get_label_row('sample_label_uid')
+
+Use the ``project.get_labels_list()`` method to get a list each label ID (``<label_hash>`` uid) in a project and conveniently fetch all project labels.
+
+.. code-block::
+
+    project = client.get_project()
+
+    label_rows = []
+    for label_hash in project.get_labels_list():
+        lb = client.get_label_row(label_hash)
+        label_rows.append(lb)
+
+The label row object contains data units with signed URL's (``data_link``) to the labeled data asset.
```

Comment:
```suggestion
The label row object contains data points with signed URLs (``data_link``: string) to the labeled data asset.
```

Reply:
I think that data unit is used to often that we cannot start calling it data points :-)